### PR TITLE
Add MATH+ questionset

### PR DESCRIPTION
**Pull request by Metamost**
This is the question set used in the DFG Excellence Cluster The Berlin Mathematics Research Center MATH+ (EXC-2046/1, project ID: 390685689)
It was derived from the original RDMO questionnaire and reduced so that it takes no longer than 20 minutes to answer. Nevertheless, it maps the recommendation for handling research data from the DFG homepage.: https://www.dfg.de/en/research_funding/principles_dfg_funding/research_data/index.html

### CODE REVIEW

**Comment by jochenklar** [comment]
Hi @Metamost ,

One thing: `https://rdmorganiser.github.io/terms/domain/project/dataset/sharing/uri` does not exist (yet), there is `https://rdmorganiser.github.io/terms/domain/project/dataset/storage/uri` and `https://rdmorganiser.github.io/terms/domain/project/dataset/uri` though.

Otherwise, the import went fine!

**Review by jochenklar — APPROVED** [review]

**Comment by jochenklar** [comment]
Looks good, @triole @Zack-83 please have a look.

File: shared/MATH+/mathplus_questions.xml [hunk 1]
```diff
@@ -0,0 +1,1733 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<rdmo xmlns:dc="http://purl.org/dc/elements/1.1/" created="2022-11-07T19:25:51.488056+01:00">
+	<catalog dc:uri="https://rdmo.mathplus.de/terms/questions/mathplus">
+		<uri_prefix>https://rdmo.mathplus.de/terms</uri_prefix>
+		<key>mathplus</key>
+		<dc:comment>Dieser Fragenkatalog ist eine Anpassung an die Erfordernisse der mathematischen DMP im Umfeld der Berliner Mathematik. Er basiert in großen Teilen auf dem DFG Fragenkatalog des Projekts Forschungsdaten Katalog (https://github.com/rdmorganiser/rdmo-catalog/tree/master/shared/fodako) Er ist Teilmenge des Katalogs "RDMO" und enthält nur die Fragen, die die Anforderungen in den "Leitlinien zum Umgang mit Forschungsdaten" der DFG, http://www.dfg.de/download/pdf/foerderung/antragstellung/forschungsdaten, abdecken. Die Fragen sind somit ausgesucht für Datenmanagementpläne, die Förderanträgen an die Deutsche Forschungsgemeinschaft (DFG) beigelegt werden sollen und keine fachspezifischen Zusatzanforderungen zu erfüllen haben.</dc:comment>
```
Comment (Zack-83) [inline shared/MATH+/mathplus_questions.xml:6]:
```suggestion
		<dc:comment>Dieser Fragenkatalog ist eine Anpassung an die Erfordernisse der mathematischen DMP im Umfeld der Berliner Mathematik. Er basiert in großen Teilen auf dem DFG-Fragenkatalog aus dem Projekt FoDaKo - Forschungsdatenmanagement und digitale Langzeitarchivierung in Kooperation (https://github.com/rdmorganiser/rdmo-catalog/tree/master/shared/fodako). Er ist eine Teilmenge des Katalogs "RDMO" und enthält nur die Fragen, die die Anforderungen in den "Leitlinien zum Umgang mit Forschungsdaten" der DFG, http://www.dfg.de/download/pdf/foerderung/antragstellung/forschungsdaten, abdecken. Die Fragen sind somit ausgesucht für Datenmanagementpläne, die Förderanträgen an die Deutsche Forschungsgemeinschaft (DFG) beigelegt werden sollen und keine fachspezifischen Zusatzanforderungen zu erfüllen haben.</dc:comment>
```

File: shared/MATH+/mathplus_questions.xml [hunk 2]
```diff
@@ -0,0 +1,1733 @@
...
+		<dc:comment/>
+		<attribute dc:uri="https://rdmorganiser.github.io/terms/domain/project/additional_rdm_policy/yesno"/>
+		<questionset dc:uri="https://rdmo.mathplus.de/terms/questions/mathplus/general/other-requirements-yesno"/>
+		<is_collection>False</is_collection>
+		<is_optional>False</is_optional>
+		<order>0</order>
+		<help lang="en">One example of discipline-specific requirements: 
+
+- &lt;a href="http://www.dfg.de/download/pdf/foerderung/antragstellung/forschungsdaten/guidelines_biodiversity_research.pdf" target=_blank"&gt;Guidelines on the Handling of Research
+                                Data in Biodiversity Research&lt;/a&gt;
```
Comment (Zack-83) [inline shared/MATH+/mathplus_questions.xml:56]:
```suggestion
- &lt;a href="http://www.dfg.de/download/pdf/foerderung/antragstellung/forschungsdaten/guidelines_biodiversity_research.pdf" target=_blank"&gt;Guidelines on the Handling of Research Data in Biodiversity Research&lt;/a&gt;
```

**Comment by Zack-83** [comment]
Looks good. The two conditions might be unified in only one file or even integrated into the main conditions file

**Comment by Metamost** [comment]
Unifying the two conditions is absolutely fine. What is the main conditions file? If that means integrating it in the official RDMO conditions file I would be delighted. 
And may I suggest to have a look at the attribute where our users can store the link or identifier to their published dataset? /MATH+/mathplus_domain_sharing_uri.xml  
It seems to me that could be of interest to the community as well. 
And I'll thank you tomorrow online for correcting the name and the unnecessary linebreak : ).

**Comment by Zack-83** [comment]
> Unifying the two conditions is absolutely fine. What is the main conditions file? If that means integrating it in the official RDMO conditions file I would be delighted. And may I suggest to have a look at the attribute where our users can store the link or identifier to their published dataset? /MATH+/mathplus_domain_sharing_uri.xml It seems to me that could be of interest to the community as well. And I'll thank you tomorrow online for correcting the name and the unnecessary linebreak : ).

The main conditions file is https://github.com/rdmorganiser/rdmo-catalog/blob/master/rdmorganiser/conditions/rdmo.xml

To integrate your creations into the main RDMO files we should get approval by the community. We can achieve it the latest by the next meeting of the _UAG Redaktionsprozesse_, the 24.11.
You're welcome to join the discussion here: https://rdmo.slack.com/archives/C03GYKPR98C/p1668154257049909

**Review by Zack-83 — CHANGES_REQUESTED** [review]
Hallo Tim, wie besprochen hier meine Empfehlung:

- Attribut & Bedingung ins zentrale Repo schieben #164
- Deinen Fragenset anpassen und die drei überflüssigen Dateien löschen
- Merge Request wird ausgeführt

**Review by Zack-83 — APPROVED** [review]

**Comment by Zack-83** [comment]
Gut gemacht!

**Pull request merged**